src/style.scss

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+}
+
+.card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__icon {
+    display: flex;
+    padding: 32px 19px 40px;
+    img {
```

Comment:
do not use selectors by tag, [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) rule 1